packages/remix-eslint-config/node.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @see https://github.com/eslint/eslint/issues/3458
+ * @see https://www.npmjs.com/package/@rushstack/eslint-patch
+ */
+require("@rushstack/eslint-patch/modern-module-resolution");
+
+module.exports = {
+  plugins: ["node"],
```

Comment:
I think it doesn't make sense to have `eslint-plugin-node` in the `plugins` array if we're enabling any rules.

So I think we should either enable some basic rules **or** we should remove it from the `plugins` array